ansible/roles/pre-ansible/tasks/coreos_bootstrap.yml

```diff
@@ -0,0 +1,36 @@
+- name: CoreOS | Create bootstrap directory
+  raw: mkdir -p {{ bootstrap_script_dir }}
+
+- name: CoreOS | Check if we need to download pypy source
+  raw: stat /tmp/pypy-{{ pypy_version }}-linux64.tar.bz2 
+  register: need_pypy_tar
```

Comment:
I don't see where pypy_version is being set. It was PYPY_VERSION=2.4.0 in bootstrap.sh

I believe we should add the following to ansible/roles/pre-ansible/defaults/main.yml:
```
pypy_version: 2.4.0
```